src/Applications/GEOSdas_App/GEOSdas.csm

```diff
@@ -5695,6 +5646,13 @@ endif
               /bin/cp $EXPID.ana_radstat_rst.$rtag.tar $RSTHOLD/ &
             endif
 
+        # hold lfo files for the next segment of ldas coupling
+        #----------------------------------------------------------
+           if  ( $LDAS_ANA ) then
```

Comment:
It looks like the indentation here is messed up whereas it looks ok in the surrounding code.  This should be fixed.

Reply:
@gmao-rreichle 
I will make it a task to comply with the indentation convention and do one commit for all.